From d7885e6a737866ada8445fd86c6352ac5ba96d7d Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Tue, 4 Apr 2017 16:26:50 +0200
Subject: [PATCH] tc: m_xt: Drop needless parentheses from #if checks

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1326726
Upstream Status: iproute2.git commit a05b9557f48e2

commit a05b9557f48e23e69a4c1a597f2f288f133ef170
Author: Phil Sutter <phil@nwl.cc>
Date:   Fri Jan 13 13:06:19 2017 +0100

    tc: m_xt: Drop needless parentheses from #if checks

    Signed-off-by: Phil Sutter <phil@nwl.cc>
---
 tc/m_xt.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/tc/m_xt.c b/tc/m_xt.c
index 890dddd..478523d 100644
--- a/tc/m_xt.c
+++ b/tc/m_xt.c
@@ -77,7 +77,7 @@ static struct xtables_globals tcipt_globals = {
 	.orig_opts = original_opts,
 	.opts = original_opts,
 	.exit_err = NULL,
-#if (XTABLES_VERSION_CODE >= 11)
+#if XTABLES_VERSION_CODE >= 11
 	.compat_rev = xtables_compatible_revision,
 #endif
 };
@@ -126,7 +126,7 @@ static int get_xtables_target_opts(struct xtables_globals *globals,
 {
 	struct option *opts;
 
-#if (XTABLES_VERSION_CODE >= 6)
+#if XTABLES_VERSION_CODE >= 6
 	opts = xtables_options_xfrm(globals->orig_opts,
 				    globals->opts,
 				    m->x6_options,
@@ -204,7 +204,7 @@ static int parse_ipt(struct action_util *a, int *argc_p,
 			break;
 
 		default:
-#if (XTABLES_VERSION_CODE >= 6)
+#if XTABLES_VERSION_CODE >= 6
 			if (m != NULL && m->x6_parse != NULL) {
 				xtables_option_tpcall(c, argv, 0, m, NULL);
 #else
@@ -242,7 +242,7 @@ static int parse_ipt(struct action_util *a, int *argc_p,
 	}
 
 	/* check that we passed the correct parameters to the target */
-#if (XTABLES_VERSION_CODE >= 6)
+#if XTABLES_VERSION_CODE >= 6
 	if (m)
 		xtables_option_tfcall(m);
 #else
-- 
1.8.3.1

